boards/arm/arduino_opta_m4/board.cmake

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(dfu-util "--pid=2341:0364" "--alt=0" "--dfuse")
+
+include(${ZEPHYR_BASE}/boards/common/dfu-util.board.cmake)
```

Comment:
does this actually work? how do you load the firmware on the board?

Reply:
Yes all the stm32h7 has a ROM DFU bootloader ( also the Arduino opta bootloader is DFU ), to load the firmware just connect the board to the host computer, hit the reset button shortly two times to make the Opta in DFU mode, then issue west flash.

This procedure is covered into the documentation, it is the same to other stm32h7 arduino boards :)

Reply:
Thanks, just checking, I'm unfamiliar with the platform. :-)